x-pack/plugins/stack_connectors/server/connector_types/cases/jira/service.test.ts

```diff
@@ -443,6 +443,70 @@ describe('Jira service', () => {
       });
     });
 
+    test('escapes special characters from summary', async () => {
```

Comment:
This tests the `createIncident` function. We need to test the `getIssues` function.  Something like this:

```
test('it should escape JQL special characters', async () => {
      requestMock.mockImplementation(() =>
        createAxiosResponse({
          data: {
            issues: issuesResponse,
          },
        })
      );

      await service.getIssues('[th!s^is()a-te+st-{~is*s|ue?or&and\\bye:}]');
      expect(requestMock).toHaveBeenLastCalledWith({
        axios,
        logger,
        method: 'get',
        configurationUtilities,
        url: <the_url_escaped>,
      });
    });
```

Reply:
cool, will remove it

Reply:
removed